### PR TITLE
Rhornung67/task/tut improve

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This PR contains a variety of content enhancements and fixes (grammar, spelling, code references, etc.) in the "Simple Loops and Basic RAJA Features" section of the RAJA User Guide tutorial.  The RTD generated documentation for this PR is here: https://raja.readthedocs.io/en/rhornung67-task-tut-improve/sphinx/user_guide/tutorial.html

### CODE REVIEW

**Comment by rhornung67** [comment]
@mdavis36 @rchen20 @MrBurmark could I get some of your eyeballs to look this over and review?

File: docs/sphinx/user_guide/tutorial.rst [hunk 1]
```diff
@@ -158,8 +169,9 @@ with respect to RAJA usage. We describe them here.
    by-value can help avoid incorrect CPU code since the compiler will report 
    incorrect usage.
 
+|br|
 
- * **Must use 'device' annotation for CUDA device execution.** 
+ * **The  'device' annotation s required for CUDA device execution.**
```
Comment (MrBurmark) [inline docs/sphinx/user_guide/tutorial.rst:174]:
```suggestion
 * **The  'device' annotation is required for CUDA device execution.** 
```

File: docs/sphinx/user_guide/tutorial.rst [hunk 2]
```diff
@@ -158,8 +169,9 @@ with respect to RAJA usage. We describe them here.
    by-value can help avoid incorrect CPU code since the compiler will report 
    incorrect usage.
 
+|br|
 
- * **Must use 'device' annotation for CUDA device execution.** 
+ * **The  'device' annotation is required for CUDA device execution.**
```
Comment (MrBurmark) [inline docs/sphinx/user_guide/tutorial.rst:174]: Should we mention CUDA and HIP?

**Review by MrBurmark — APPROVED** [review]

File: docs/sphinx/user_guide/tutorial.rst (hunk 2 above)
Comment (rhornung67) [reply docs/sphinx/user_guide/tutorial.rst]: Yup. Pushed a change. Thank you.

**Review by mdavis36 — APPROVED** [review]

**Comment by rhornung67** [comment]
@mdavis36 is this worth merging, if we're reworking the material anyway?

**Comment by rhornung67** [comment]
@mdavis36 is this worth merging, if we're reworking the material anyway?

**Comment by mdavis36** [comment]
> @mdavis36 is this worth merging, if we're reworking the material anyway?

I think so, my branch is working off of this PR.

**Pull request merged**